btr-web/btr-main-app/components/person-info/Citizenship.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="w-2/3 mr-2" :class="countries.length === 1 ? 'flex justify-center' : 'grid gap-x-1 grid-cols-2'">
+    <UTooltip
+      v-for="elem in countries"
+      :key="elem.alpha_2"
+      :text="getName(elem.alpha_2) || elem.alpha_2"
+      :popper="{ arrow: true, placement: 'top' }"
+    >
+      <CountryFlag
+        :country="elem.alpha_2"
+        size="normal"
+        style="margin-top: -5px"
+      />
+    </UTooltip>
```

Comment:
Canadian Permanent Resident has a special alpha_2 "CA_PR" and it won't be recognized by `<CountryFlag>` and the `getName` function. The Canadian flag should be rendered for both "CA" and "CA_PR". The tooltip text for "CA" should be "Canada (Citizen)". For "CA_PR" the tooltip should show "Canada (Permanent Resident)".

Reply:
Good catch, changed.